### PR TITLE
[Components] recruitis #11523

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

**Comment by lcaresia** [comment]
@luancazarine For some reason script don't assigned an reviewer, so I assigned you manually.

**Review by luancazarine — DISMISSED** [review]
Hi @lcaresia, thanks for your contribution! LGTM!

File: components/recruitis/actions/create-answer/create-answer.mjs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { ConfigurationError } from "@pipedream/platform";
+import app from "../../recruitis.app.mjs";
+
+export default {
+  key: "recruitis-create-answer",
+  name: "Create Answer",
+  description: "Create a new answer for a job. [See the documentation](https://docs.recruitis.io/api/#tag/Candidates/paths/~1answers/post)",
+  version: "0.0.{{ts}}",
```
Comment (luancazarine) [inline components/recruitis/actions/create-answer/create-answer.mjs:8]:
```suggestion
  version: "0.0.1",
```

**Review by luancazarine — CHANGES_REQUESTED** [review]
Hi @lcaresia, I just added a minor change request.

**Review by luancazarine — APPROVED** [review]

**Pull request merged**